prototypes/toggle-controls/generate-reader.py

```diff
@@ -3,91 +3,60 @@
 import os
 import shutil
 
-from pysblgnt import morphgnt_rows
+from jinja2 import Environment, FileSystemLoader
 
+from utils import rows_by_verses_by_chapters_for_book
 
-def generate(book_title, book_num, output_filename):
+
+env = Environment(
+    loader=FileSystemLoader("."),
+)
+template = env.get_template("template.html")
+
+
+def before(row):
+    word = row["word"]
+    text = row["text"]
+    return text[:text.index(word)]
+
+
+def after(row):
```

Comment:
I was wondering yesterday if `before` and `after` could be implemented with `split(word, 1)`. It wouldn't be ideal for how this code is separated, but it struck me wondering if we reimplemented split to an extent ;-)